internal/oracle/oracle_test.go

```diff
@@ -362,10 +383,20 @@ bar = 7`
 					t.Fatal(err)
 				}
 			}
+			buffer := tc.modules["buffer.rego"]
+			before := tc.pos - 4
+			if before < 0 {
+				before = 0
+			}
+			after := tc.pos + 5
+			if after > len(buffer) {
+				after = len(buffer)
+			}
+			t.Logf("pos is %d: \"%s<%s>%s\"", tc.pos, string(buffer[before:tc.pos]), string(buffer[tc.pos]), string(buffer[tc.pos+1:after]))
```

Comment:
ℹ️ This is a help output for test authoring -- I really need this to figure out which position I want to lookup. Figured I'd keep it.